examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>The next cell (implementation) should be introduced by the text. For example, that it should be an implementation of the above formula using just Aesara operations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I find confusing that it uses <code>np.log</code>  and <code>tt.switch</code>  <span class="ql-cursor">﻿</span></p>